### PR TITLE
correct china standard chart of account.

### PR DESCRIPTION
**Pull request by jeffery9**

### CODE REVIEW

**Comment by jeffery9** [comment]
@mart-e  this is some fix for china account of chart. please merge it. thanks.

**Comment by elicoidal** [comment]
@jeffery9 Good start.
Can you add the default currency to be CNY?
Can you add the missing VAT and Business taxes?

File: addons/l10n_cn_small_business/account_chart_template.xml [hunk 1]
```diff
@@ -537,7 +537,7 @@ http://kjs.mof.gov.cn/zhengwuxinxi/zhengcefabu/201111/t20111107_605525.html
             <field name="property_account_expense_categ_id" ref="small_business_chart1401"/>
             <field name="property_account_income_categ_id" ref="small_business_chart5001"/>
             <field name="income_currency_exchange_account_id" ref="small_business_chart5301"/>
-            <field name="expense_currency_exchange_account_id" ref="small_business_chart5712"/>
+            <field name="expense_currency_exchange_account_id" ref="small_business_chart5711"/>
```
Comment (cysnake4713) [inline addons/l10n_cn_small_business/account_chart_template.xml]: income currency exchange and expense currency exchange should use 'exchange loss' under 'Financial expenses',
-------------------------------
兑汇用科目,应该在财务费用科目中下设子科目'兑汇损益'专门做这事,不应该放在营业里面

**Comment by cysnake4713** [comment]
'transfer_account_id' should use '' 'In transit funding' account under 'Other monetary fund' account.

-------------------
会计中,银行间账目周转的,应该使用'其它货币资金'下设的子科目'在途资金'来做这事

**Comment by jeffery9** [comment]
汇兑收益和汇兑损失， 在 小企业会计准则 和 企业会计准则 里面的规定是不同的。

小企业    汇兑收益，记在 “营业外收入”， 汇兑损失，记在 “财务费用”。

企业   汇兑收益，记在 “汇兑损益”， 汇兑损失，也记在 “汇兑损益”。

我再查看下


On Wed, Dec 9, 2015 at 10:28 AM, matt.cai <notifications@github.com> wrote:

> 'transfer_account_id' should use '' 'In transit funding' account under
> 'Other monetary fund' account.
> ------------------------------
>
> 会计中,银行间账目周转的,应该使用'其它货币资金'下设的子科目'在途资金'来做这事
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/odoo/odoo/pull/9940#issuecomment-163086814>.
>



-- 
Jeffery        -odoo expert form kunshan, suzhou, china

**Comment by mart-e** [comment]
I have no idea what you are talking about so let me know once the PR is ready :smile:

**Comment by elicoidal** [comment]
Hehe, welcome to my world ;)

**Comment by jeffery9** [comment]
@mart-e  I think that the PR should merge to  branch master and 9.0

thank you.

On Thu, Dec 10, 2015 at 4:30 PM, Eric Caudal <notifications@github.com>
wrote:

> Hehe, welcome to my world ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/odoo/odoo/pull/9940#issuecomment-163532740>.
>



-- 
Jeffery        -odoo expert form kunshan, suzhou, china

**Comment by elicoidal** [comment]
@jeffery9
Why not include all the VAT rate in China?

**Comment by antonylesuisse** [comment]
Could you please clean up comments from the first commit "review china COA, add the lost account and correct the existing.." ?

**Comment by jeffery9** [comment]
yes, have delete it. thanks.

On Sun, Dec 27, 2015 at 7:32 AM, antonylesuisse <notifications@github.com>
wrote:

> Could you please clean up comments from the first commit "review china
> COA, add the lost account and correct the existing.." ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/odoo/odoo/pull/9940#issuecomment-167371414>.
>



-- 
Jeffery        -odoo expert form kunshan, suzhou, china

**Comment by mart-e** [comment]
@jeffery9 maybe you have forgotten to push your changes but for instance the template `chart1003` (存放中央银行款项) is still commented

**Comment by jeffery9** [comment]
#12581

**Comment by tivisse** [comment]
Hi @jeffery9 

Is this PR still relevant?

Thanks.

**Pull request closed without merging**